governance.md

```diff
@@ -43,7 +43,6 @@ with the following tips:
 
 **Dos**
 
-- Do check on IRC if anyone can give you advice or backstory on the issue
```

Comment:
Should we replace with something about the forums?

Reply:
Yeah, probably. Will add a new sentence.